detections/endpoint/windows_modify_registry_for_ie_proxy_settings.yml

```diff
@@ -0,0 +1,101 @@
+name: Windows Modify Registry for IE Proxy Settings
+id: d833f972-1ff4-4ef6-b020-831aa6d13fdd
+version: 2
+date: '2025-07-15'
+author: Zaki Zarkasih (FPT Metrodata Indonesia)
+type: TTP
+status: production
+description: >
+  This detection identifies suspicious modifications to Internet Explorer (IE) proxy configuration via Windows Registry. 
+  Adversaries commonly abuse IE proxy settings to redirect network traffic through attacker-controlled infrastructure, often 
+  by modifying keys such as `ProxyEnable`, `ProxyServer`, `AutoConfigURL`, or even `ProtocolDefaults` and `ZoneMapDomains` 
+  to affect how Internet traffic is routed. These changes are typically made using command-line utilities such as `reg.exe`, `powershell.exe`, or `cmd.exe`. Such 
+  behavior may indicate attempts to bypass security controls, establish covert C2 channels, or manipulate user traffic for credential interception.
+data_source:
+  - Windows Event Log Security 4688
+search: >
+  index="windows"
+  sourcetype=WinEventLog:Security
+  EventCode=4688
+  Process_Command_Line="*\\Software\\Microsoft\\Windows\\CurrentVersion\\Internet Settings*"
+  (New_Process_Name="*\\reg.exe" OR New_Process_Name="*\\powershell.exe" OR New_Process_Name="*\\cmd.exe")
+  NOT (
+      Details="DWORD*" 
+      OR Details IN ("Cookie:", "Visited:", "(Empty)") 
+      OR match(Process_Command_Line, ".*\\\\Cache.*|.*\\\\ZoneMap.*|.*\\\\WpadDecision.*") 
+      OR Details="Binary Data" 
+      OR Process_Command_Line="*\\Software\\Microsoft\\Windows\\CurrentVersion\\Internet Settings\\Accepted Documents*" 
+      OR New_Process_Name="C:\\Windows\\System32\\RuntimeBroker.exe"
+  )
+  | rex field=Process_Command_Line "(?i)(new-itemproperty|set-itemproperty|set-item|reg\\s+add|new-item)[^\n\r]*?(?<registry_path>HK(?:EY_)?(?:LM|CU|CR|U)[:\\\\][^\"'\s]+(?:\\\\[^\"'\s]*)*)"
```

Comment:
The details field is not part of a 4688 event

Also after thinking about it, the regex is overkill.

While it is nice to have that extraction for better display. I do not think it will add a lot of value. Since the commandline have to be inspected regardless.